### ISSUE TITLE
Outstanding minor fixes

### ISSUE THREAD

**Issue by veronicar239** [pimbongaerts/mesophotic#58]
- [x] Publication metadata:  section "Details for Scientific (Review) Articles". A blank selection is needed under the dropdown menu for the "Journal" category. Technical reports do not have an associated journal, but at the moment each technical report is automatically (mistakenly) assigned the first alphabetically listed journal (Acta Ichthyol). For example see http://mesophotic.org/publications/281/edit
- This is also an issue for book chapters. Book chapters are automatically assigned the first alphabetically-listed journal (Acta Ichtyol). For example see http://mesophotic.org/publications/270/

- [x] After this is fixed, we need to go through and edit (remove erroneous 'Acta Ichtyol'):
- technical reports
- book chapters
- technical books
- scientific books

**Comment by veronicar239** [comment]
@pimbongaerts  
For purpose of stats page, create new tag "Open circuit" to encompass current Platform tags 'Diving unspecified' and 'Diving open circuit' (and perhaps 'Diving technical' as well - to be decided). This will then be used to contrast between open circuit and 'Diving rebreather' tag for the platform types in the summary statistics page.

**Comment by veronicar239** [comment]
@pimbongaerts 
Upload downloadable Rmarkdown file (for summary statistics) to Downloads page.

**Comment by veronicar239** [comment]
@pimbongaerts 
The new Behind the Science post (http://www.mesophotic.org/posts/43 added on 28-3-19) is not showing on the main page http://mesophotic.org/posts. Further, I cannot find the post on any of the three pages of BTS posts (when I use control/find to look for the author's last name or keyword of the title). 

1) Can you check that new posts are automatically added to the top of the 'posts' page?
2) Can you also check that all archived posts are stored within the multiple pages of mesophotic.org/posts?

**Comment by veronicar239** [comment]
I found a random publication that shows text that is 'center-aligned' rather than the usual 'left-hand-aligned'. This occurs both on the publication page as well as on the publication/edit page.  http://mesophotic.org/publications/352 
http://mesophotic.org/publications/352/edit

**Comment by pimbongaerts** [comment]
Fixed with dae73be

**Comment by pimbongaerts** [comment]
Removed erroneous references to Acta etc through a script:

```
	Publication.where(journal_id: 145).each do |publication|
		if not publication.scientific_article?
			puts "Removing journal for #{publication.id}:#{publication.title}"
			publication.journal = nil
			publication.save!
		end
	end
```
